release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -480,8 +480,8 @@ For more information, see xref:../machine_management/applying-autoscaling.adoc#c
 This release introduces the ability to manage machines by using the upstream Cluster API, integrated into {product-title}, as a Technology Preview for {vmw-full} clusters.
 This capability is in addition or an alternative to managing machines with the Machine API.
 For more information, see _Managing machines with the Cluster API_.
-//xref blocked by #72885
-//For more information, see xr3f:../machine_management/cluster_api_machine_management/cluster-api-about.adoc#cluster-api-about[Managing machines with the Cluster API].
+
+For more information, see xref:../machine_management/cluster_api_machine_management/cluster-api-about.adoc#cluster-api-about[Managing machines with the Cluster API].
```

Comment:
```suggestion
For more information, see xref:../machine_management/cluster_api_machine_management/cluster-api-about.adoc#cluster-api-about[About the Cluster API].
```
IDK if you want to change this text to match the page we're going to. I was confused for a sec. Users might be, too?

Reply:
sure :) thanks!